### DISCUSSION TITLE
Metaplex Staking

### DISCUSSION THREAD

**Discussion by svc01-metaplex** [metaplex-foundation/mip#35] [MIP Submissions]
Submitter: Charlesfoodog
Email: charlessublette@gmail.com
Submitter wallet: 2YxHRtQJ914rQkxqyFYC1VxfpJWMZHLqc7SAMPVjipRv
Submission transaction: 2ErdiyiYZPiBpRNJUyW6BUBRn8txmvrNBcyDpAMJjiRzourwzxZZ5gbyKQAa9wgXw4VmtWYZWbNAyrAXqVErkqiR
Title: Metaplex Staking  
Links: https://www.metaplex.com/

Metaplex will build functionality for their token to be staked.

Using 100,000,000 of the Metaplex DAO tokens. (160,000,000 total) to be put into staking rewards that will allow anyone to stake their Metaplex DAO tokens and earn staking rewards.  The emission rate would be 10% per year until the pool is drained.  Metaplex DAO tokens can be unstaked at any time. However, only people who have staked for a minimum of 72 hours -post-staking event can begin receiving rewards.

Metaplex will create the staking mechanism that will allow anyone to stake their tokens from the pool and maintain all of the functionality, Contracts, Audits, and anything else for the duration of the pool being active.  The staking portal will be added to Metaplex's homepage on the navigation bar.   www.metaplex.com

**Comment by MetaplexFoundation** [comment]
Thanks for the submission, Charlesfoodog. MIP submissions are limited to proposals for changes to the Metaplex Program Library. Requests for allocations of tokens from the Metaplex DAO treasury should be made via MPLX grant proposals. To submit an MPLX grant proposal, please see here: https://mip.metaplex.com/ - Accordingly, this MIP submission is being screened out. 

Proposals for building new programs and tools in the Metaplex Program Library, in this case a staking contract, may be made via MIP submissions. However, such submissions must include critical details necessary for community evaluation. Please see here for the MIP guidelines (https://github.com/metaplex-foundation/mip/blob/main/mip-5.md).
